docs/_docs/Unofficial Sidecar Academy/Sidecar-Academy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: LDSA Unofficial Sidecar Academy
+category: Unofficial Sidecar Academy
+order: 1
+---
+
+Contents:
+- [Course Overview](#course-overview)
+- [What will be covered](#what-will-be-covered)
+- [What is the same and what is different](#what-is-the-same-and-what-isdifferent)
+
+---
+
+# Course overview
+Over the last few years, many people have wanted to but not been able to get into the LDSSA. As of now, we have not been able to offer them any community-based alternatives which leaves them with the usual options of lonely online courses. While the course material, portal, and access to instructors are definitely useful things, the vast majority of the value provided to the students is in the community aspect. The feeling that you're not going it alone is incredibly powerful.
+
+Since 2020, we have started to offer the LDSA Unofficial Sidecar Academy, a community-based lightweight version of the LDSA and is meant to provide all of those who didn't have a chance to get into the 2020 LDSSA, a concrete next step in their pursuit of Data Science.
+
+## What will be covered
+
+The Sidecar Academy consists of (1) [Data Science Prep Course](https://ldssa.github.io/wiki/DS%20Prep%20Course/Data-Science-Prep-Course/) and (2) [Lisbon Data Science Starters Academy (LDSSA)](https://ldssa.github.io/wiki/Starters%20Academy%20(LDSSA)/01-Starters-Academy-(Course)/)
+
+Proficiency in Python, and basic knowledge of Software engineering is required to adequately follow LDSSA's learning materials. As such, we will first cover the Prep Course to provide you with Python skills and other requirements. We'll then go through the LDSSA.
+
+## What is the same and what is different?
+
+The core material is the same. Since the LDSA material is all open-source under the very permissive MIT license, anyone is free to use the course material. And this is what the Sidecar Academy will do. All of the learning units, all of the exercises, all of the recorded lectures will be 100% the same.
+
+The differences are the following:
+
+1. There will be a different community. Many of the instructors that are available on the LDSSA will not be directly available in the Sidecar Academy.
+
+2. The students will need to help each other a LOT. Since there will probably be less instructors available, this is a necessity.
+
+3. There will not be a grading portal. In the LDSA we normally use a grading portal to evaluate the progress of all of the students. This requires some specialized work which is not available for the Sidecar Academy.
+
+4. It will be completely free.
+
+5. Hackathons will follow a format similar to Kaggle challenges (More details in section below)
+
+
+### Hackathons 
+
+TO-DO
+
+
+### Introduction session
+
+TO-DO
```

Comment:
I'm not sure what is supposed to be in this section, but I think we're missing a section explaining what are the materials and how they work like we have here -> https://github.com/LDSSA/sidecar-academy/wiki/USA-Course#learning-units